search-website-functions-v4/api/Lookup.cs

```diff
@@ -0,0 +1,56 @@
+using System.Net;
+using Azure.Search.Documents.Models;
+using Azure.Search.Documents;
+using Azure;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using WebSearch.Models;
+using Azure.Core;
+
+namespace WebSearch.Function
+{
+    public class Lookup
+    {
+        private static string searchApiKey = Environment.GetEnvironmentVariable("SearchApiKey", EnvironmentVariableTarget.Process);
+        private static string searchServiceName = Environment.GetEnvironmentVariable("SearchServiceName", EnvironmentVariableTarget.Process);
+        private static string searchIndexName = Environment.GetEnvironmentVariable("SearchIndexName", EnvironmentVariableTarget.Process) ?? "good-books";
+
+        private readonly ILogger _logger;
+
+        public Lookup(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<Lookup>();
+        }
+
+        [Function("lookup")]
+        public async Task<HttpResponseData> RunAsync([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req, FunctionContext executionContext)

```

Comment:
```suggestion
        public async Task<HttpResponseData> RunAsync(
            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req, 
            FunctionContext executionContext)
```

Minor formatting nit, whenever you can, try to format to avoid horizontal scrolling.